presidio-api/cmd/presidio-api/docs/swagger.yaml

```diff
@@ -0,0 +1,1890 @@
+basePath: /api/v1
+consumes:
+- application/json
+definitions:
+  AnalyzeRequest:
+    description: AnalyzeRequest represents the request to the analyze service via
+      GRPC
+    properties:
+      analyzeTemplate:
+        $ref: '#/definitions/AnalyzeTemplate'
+      text:
```

Comment:
Optional property. Is it actually optional, i.e. can we call AnalyzeRequest without a text and it does something meaningful? If not, can we make it mandatory?

Reply:
I think it could be mandatory, or stay optional (if the text is blank, the response will be blank). 
Anyway, if decided to change to mandatory, the change must be done in the [presidio-genproto](https://github.com/Microsoft/presidio-genproto) repository.